### DISCUSSION TITLE
Version 4.7

### DISCUSSION THREAD

**Discussion by joakimeriksson** [contiki-ng/contiki-ng#1614] [General]
# Press release
We are releasing version 4.7 of Contiki-NG. This release adds several vulnerability fixes and stability improvements. It also adds: support for the nRF5340 platform and CPU (Cortex M33), with TSCH support; Cleaner IP packet RSSI/LQI API; improved neighbor table management and many other improvements.

# Changelog
- Several security-related bugfixes (#1482,#1506,#1517, #1541,#1589)
- Added support for nRF5340 (M33, dual core)
- Added DHT-11 sensor driver
- Introduced the MAC protocol return value MAC_TX_QUEUE_FULL
- Added RSSI and LQI as uipbuf attributes (and deprecated the RSSI API in 6LoWPAN layer)
- Several TSCH improvements, including an optimized Orchestra rule for the root node
- Improved neighbor table replacement policy
- Migrated most of the Python code to Python 3
- Added MSPMote clock deviation in Cooja simulations (and several other Cooja improvements)
- Lots of other minor fixes and additions

<hr /><em>This discussion was created from the release <a href='https://github.com/contiki-ng/contiki-ng/releases/tag/release/v4.7'>Version 4.7</a>.</em>

**Comment by ahmed1galal** [comment]
HI All,
I needed help , if possible, in the contiki program when run ipv6
![Screenshot from 2021-08-15 11-07-08](https://user-images.githubusercontent.com/88966052/129473990-74bc1156-fe28-418c-b1df-26c6c17d6359.png)

**Reply by atiselsts** [reply]
Hi @ahmed1galal , this help request is off-topic for this discussion. Further, we only support Contiki-NG here.
